Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.14-alpine
+
+ARG version
+ENV VERSION=${version}}
+
+RUN mkdir /app
+ADD . /app
+WORKDIR /app
+
+COPY go.mod go.sum ./
+
+RUN go mod download
+
+RUN go build -o main ./cmd/
```

Comment:
Is it normal to build the executable in the container like this, thus the resulting image containing the go toolchain?